### ISSUE TITLE
Add an option to remove tracing data

### ISSUE THREAD

**Issue by 0v1se** [openethereum/openethereum#9716]
- **Parity Ethereum version**: 2.0.6
- **Operating system**: Linux
- **Installation**: built from source
- **Fully synchronized**: yes
- **Network**: ethereum
- **Restarted**: yes

Please, consider adding an option to remove old tracing information.
We use this information to build some indexes on blockchain data, but then we don't need it.
This option can be useful for many projects which process incoming new blocks via tracing api and transform this data to some other form

Thanks

**Comment by 0v1se** [comment]
For those who want to do this I implemented this feature in this branch:
https://github.com/daonomic/parity-ethereum/tree/remove_traces
It's implemented as rpc method "trace_removeBlock" - takes block number

Unfortunately, I can't create PR based on my commit because I also deleted method "trace_filter" - I don't need it and bloom filter for it occupies pretty much space.

So, if you want to have slightly modified stable parity with traces and possibility to delete those traces - use https://github.com/daonomic/parity-ethereum/tree/stable and build it from source

Currently, there is 2.0.8 version. I will try to keep this branch synced with stable branch of this repository.
It's working pretty good on Kovan for us. We're planning to move this feature to mainnet soon.
